pkg/cfn/builder_test.go

```diff
@@ -19,42 +19,42 @@ func TestBuilderAndComposers(t *testing.T) {
 		{
 			name:     "VPC composer",
 			golden:   "vpc-cloudformation.yaml",
-			composer: components.NewVPCComposer("test", "test", "192.168.0.0/20", "eu-west-1"),
+			composer: components.NewVPCComposer("test", "192.168.0.0/20", "eu-west-1"),
 		},
 		{
 			name:     "Minimal VPC composer",
 			golden:   "vpc-minimal-cf.yaml",
-			composer: components.NewMinimalVPCComposer("test", "test", "192.168.0.0/20", "eu-west-1"),
+			composer: components.NewMinimalVPCComposer("test", "192.168.0.0/20", "eu-west-1"),
 		},
 		{
 			name:     "ExternalSecretsPolicy composer",
 			golden:   "esp-cloudformation.yaml",
-			composer: components.NewExternalSecretsPolicyComposer("repo", "test"),
+			composer: components.NewExternalSecretsPolicyComposer("test"),
 		},
 		{
 			name:     "AlbIngressControllerPolicy composer",
 			golden:   "alb-ingress-cloudformation.yaml",
-			composer: components.NewAlbIngressControllerPolicyComposer("repo", "env"),
+			composer: components.NewAlbIngressControllerPolicyComposer("tset"),
```

Comment:
tset -> test?

I don't care about THE spelling, I'm just mentioning in case it can cause a test bug at some point.

Reply:
Doh, it won't have any effect on anything, so we are good